src/Controller/Component/ModulesComponent.php

```diff
@@ -198,4 +199,55 @@ public function objectTypes(?bool $abstract = null) : array
 
         return $types;
     }
+
+    /**
+     * Upload a file and store it in a media stream
+     *
+     * @param BEditaClient $apiClient The api client to perform upload
+     * @param array $requestData The request data from form
+     * @return void
+     */
+    public function upload(BEditaClient $apiClient, array &$requestData) : void
```

Comment:
API client may be used with `ApiClientProvider::getApiClient()` no need to pass it as argument